package.json

```diff
@@ -35,6 +35,7 @@
     "start": "node server/index.js",
     "server": "nodemon --watch client --watch server --watch templates --ext js,jsx server/index.js",
     "server-dev": "nodemon --watch client --watch server --watch templates --ext js,jsx server/index-dev.js",
+    "server-hot": "nodemon --watch client --watch server --watch templates --ext js,jsx server/index-hot.js",
```

Comment:
I needed a hook to add a `WEBPACK_HOT` environmental variable. I followed the pattern that we've got for dev, but am not in love with the repetition here.